src/widgets/PrivacyWidget.php

```diff
@@ -85,6 +85,13 @@ class PrivacyWidget extends Widget
                         justify-content: space-between;
                     }';
 
+    /**
+     * @var string CSS class to be applied to the container
+     * Custom CSS class can be used to style the container
+     * @since 1.0.3
+     */
+    public $cssClass;
```

Comment:
What do you think about $containerCssClass instead of $cssClass? This would make it more clear.

Reply:
Makes sense to me, changed according to that. How do you treat whitespaces? (Widget output...)

Another PR will be coming, seems the tests are failing now, not resolved yet. 
(BTW: Maybe set up auto testing for PR's?)